tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/frontend/pages/AuthenticationPage.java

```diff
@@ -36,17 +37,18 @@ public AuthenticationPage(WebDriver driver) {
     }
 
     //Actions:
-    public void signInWithBceid(String userNm, String pwd) {
+    public void signInWithBceid(String userNm, String pwd) throws InterruptedException {
         wait = new WebDriverWait(driver, 90);
         wait.until(ExpectedConditions.titleIs("Government of British Columbia"));
         log.info("Waiting for the page to load...");
         userName.sendKeys(userNm);
         password.sendKeys(pwd);
+        Thread.sleep(2000L);
```

Comment:
what is L?

Reply:
represents "Long" argument for milliseconds.